pkg/oc/clusteradd/components/default-imagestreams/default_imagestreams.go

```diff
@@ -20,6 +20,18 @@ func (c *RHELImageStreamsComponentOptions) Name() string {
 }
 
 func (c *RHELImageStreamsComponentOptions) Install(dockerClient dockerhelper.Interface) error {
+	secretComponent := componentinstall.DockerConfigSecret{
```

Comment:
Oh ordering sadness.  Is this really an independent thing to install then?  Is it narrowly tailored for this?  if so, just include in this package.

Reply:
moved